src/controllers/focus-lost-controller.stories.js

```diff
@@ -0,0 +1,97 @@
+import { css, html, LitElement, render } from 'lit';
+import { repeat } from 'lit/directives/repeat.js';
+import { enhanceStoriesNames } from '../stories/lib/story-names.js';
+import { FocusLostController } from './focus-lost-controller.js';
+import docsPage from './focus-lost-controller.md';
+import '../components/cc-button/cc-button.js';
+
+class MyList extends LitElement {
+  static get properties () {
+    return {
+      elements: { type: Array },
+    };
+  }
+
+  constructor () {
+    super();
+    this.elements = [];
+
+    // eslint-disable-next-line no-new
+    new FocusLostController(this, 'div', ({ suggestedElement }) => {
```

Comment:
I'm wondering if we should warn against using very generic selectors like this within the controller documentation ?
It's ok for the story here but `querySelectorAll('div')` or `p` etc. would also query all nested `div` children in the light DOM with these tags no ?
But then again, who would query a `div` in real components ?